vhdbuilder/packer/install-dependencies.sh

```diff
@@ -74,15 +74,17 @@ cat << EOF >> ${VHD_LOGS_FILEPATH}
 EOF
 
 VNET_CNI_VERSIONS="
+1.1.2
+1.1.0
 1.0.33
-1.0.29
 "
 for VNET_CNI_VERSION in $VNET_CNI_VERSIONS; do
-    VNET_CNI_PLUGINS_URL="https://acs-mirror.azureedge.net/cni/azure-vnet-cni-linux-amd64-v${VNET_CNI_VERSION}.tgz"
+    VNET_CNI_PLUGINS_URL="https://kubernetesartifacts.azureedge.net/azure-cni/v${VNET_CNI_VERSION}/binaries/azure-vnet-cni-linux-amd64-v${VNET_CNI_VERSION}.tgz"
```

Comment:
should not change this.
acs-mirror is a documented url for aks egress lockdown customer to down the binaries , if needed.
we keep this intentionally.

acs-mirror and kuberenteartifacts are pointing to the same storage account so there is no difference.

Reply:
Strange the cdsn in front of them are using different ips so it's really not clear their the same. Curious why aks-engine would use the one that is not documented egress url list

```
pmiller@millerwork:~/gopath/src/goms.io/aks/rp/resourceprovider/server/microsoft.com$ nslookup acs-mirror.azureedge.net
Server:         172.27.192.1
Address:        172.27.192.1#53

Non-authoritative answer:
acs-mirror.azureedge.net        canonical name = acs-mirror.akstd.azureedge.net.
acs-mirror.akstd.azureedge.net  canonical name = azureedge.mdc.akamaized.net.
azureedge.mdc.akamaized.net     canonical name = a1879.dscw14.akamai.net.
Name:   a1879.dscw14.akamai.net
Address: 63.233.60.36
Name:   a1879.dscw14.akamai.net
Address: 63.233.60.234
Name:   a1879.dscw14.akamai.net
Address: 2600:1409:3800::6862:7069
Name:   a1879.dscw14.akamai.net
Address: 2600:1409:3800::6862:7071

pmiller@millerwork:~/gopath/src/goms.io/aks/rp/resourceprovider/server/microsoft.com$ nslookup kubernetesartifacts.azureedge.net
Server:         172.27.192.1
Address:        172.27.192.1#53

Non-authoritative answer:
kubernetesartifacts.azureedge.net       canonical name = kubernetesartifacts.afd.azureedge.net.
kubernetesartifacts.afd.azureedge.net   canonical name = afd.t-0001.t-msedge.net.
afd.t-0001.t-msedge.net canonical name = t-0001.t-msedge.net.
t-0001.t-msedge.net     canonical name = Edge-Prod-WSTr3.ctrl.t-0001.t-msedge.net.
Edge-Prod-WSTr3.ctrl.t-0001.t-msedge.net        canonical name = standard.t-0001.t-msedge.net.
Name:   standard.t-0001.t-msedge.net
Address: 13.107.246.10
Name:   standard.t-0001.t-msedge.net
Address: 2620:1ec:bdf::10
```

Reply:
there is a history we can chat more - but in short aks-engine/upstream has no egress promising to external customer, AKS does.